stack/tests/deployment_test.yaml

```diff
@@ -0,0 +1,24 @@
+suite: test deployment
+templates:
+  - deployment.yaml
+tests:
+  - it: should work
+    values:
+      - values.yaml
```

Comment:
Let's use `set` instead of `values` for this test so we can just get a minimal test running